src/oss/confluent.sh

```diff
@@ -1355,13 +1390,26 @@ These are the available commands:
 
     config      Configure a connector.
 
+EOF
+for ((i = 0; i < ${#dynamic_cli_commands[@]}; ++i)); do
+local command=${dynamic_cli_commands[$i]}
```

Comment:
we should properly align everything besides the actual text printed on the terminal.